crates/typst-cli/src/compile.rs

```diff
@@ -172,6 +226,11 @@ fn export_image(
     let mut storage;
 
     for (i, frame) in document.pages.iter().enumerate() {
+        // If the frame is in the cache, skip it.
+        if export_cache.is_cached(frame) {
+            continue;
+        }
```

Comment:
I was just thinking that this might be problematic if the file has been deleted in the meantime. Maybe we could check that it is still present with the correct mtime?

Reply:
I added a check that the file exist, but I did not add the modified time check as I feel like this is
a) beyond scope since we'd need to keep track of time which I feel is unnecessary complexity
b) unnecessary, if the person messes up their files, it's on them, and otherwise the file would be overwritten anyway and they would lose their modification

Unless you feel very strongly about this, I'd leave it as is